tests/light/src/axosyslog_light/testcase_parameters/testcase_parameters.py

```diff
@@ -26,6 +26,7 @@
 
 
 INSTANCE_PATH = None
+RUNNER = None
```

Comment:
I think passing the parameter through classes is the way to go. Yes, we have a `--runner` option, but what happens when we want to run a client from a docker runner, but run the server from a local runner? We cannot pinpoit one runner for all our syslog-ng instances, neither we can pinpoint one container name, as we might have multiple containers for a client, server, etc... (this change comes in a future commit)

We tackle the same problem with the `Executor` class (`SyslogNgExecutor` and `SyslogNgCtlExecutor`).